weasyprint/tests/test_draw/__init__.py

```diff
@@ -26,19 +26,56 @@
 R = as_pixel(b'\xff\x00\x00\xff')  # red
 B = as_pixel(b'\x00\x00\xff\xff')  # blue
 G = as_pixel(b'\x00\xff\x00\xff')  # lime green
-V = as_pixel(b'\xBF\x00\x40\xff')  # Average of 1*B and 3*r.
-S = as_pixel(b'\xff\x3f\x3f\xff')  # r above r above #fff
+V = as_pixel(b'\xBF\x00\x40\xff')  # Average of 1*B and 3*R.
+S = as_pixel(b'\xff\x3f\x3f\xff')  # R above R above #fff
 r = as_pixel(b'\xff\x00\x00\xff')  # red
 g = as_pixel(b'\x00\x80\x00\xff')  # half green
 b = as_pixel(b'\x00\x00\x80\xff')  # half blue
-v = as_pixel(b'\x80\x00\x80\xff')  # Average of B and r.
+v = as_pixel(b'\x80\x00\x80\xff')  # Average of B and R.
 a = as_pixel(b'\x00\x00\xfe\xff')  # JPG is lossy...
-p = as_pixel(b'\xc0\x00\x3f\xff')  # r above r above B above #fff.
+p = as_pixel(b'\xc0\x00\x3f\xff')  # R above R above B above #fff.
+
+# NOTE: "r" is not half red on purpose. In the pixel strings it has
+# better contrast with "B" than does "R". eg. "rBBBrrBrB" vs "RBBBRRBRB".
```

Comment:
@liZe Can you clarify if this is correct?

Reply:
I don't remember the reason, but it looks like a plausible explanation.